### PR TITLE
Tried to add a new job in travis

### PR DESCRIPTION
**Pull request by pnikiel**

### CODE REVIEW

**Comment by ben-farnham** [comment]
And also under CI-builds, another directory: Common. Put design.xml etc files in there

File: .travis/test_cases/test_cache_variables/Design.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ben-farnham) [inline .travis/test_cases/test_cache_variables/Design.xml]: Would be nice if these test Design files had some comments added describing _what_ they prove. I mean, if this test passes (pass == build completes successfully?) then how much does that tell us about cachevariables?

File: .travis/job_prologue.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+           apt install python-enum34 ;
```
[the remark this answers is not in the document]
Comment (pnikiel) [reply .travis/job_prologue.sh]: Yes, I created a ticket for this, for you.

File: .travis/job_prologue.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+           ln -s /usr/bin/cmake /usr/bin/cmake3 ;
```
[the remark this answers is not in the document]
Comment (pnikiel) [reply .travis/job_prologue.sh]: I created a ticket for you.

File: .travis/test_cases/test_cache_variables/Design.xml (hunk 1 above)
Comment (pnikiel) [reply .travis/test_cases/test_cache_variables/Design.xml]: Yes, I added documentation to a file called 'description.txt' which lives just along the respective Design.

**Pull request merged**